k6-tests/subgraph/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "performance-testing-test",
+  "version": "0.0.1",
+  "scripts": {
+    "prepare:env": "yarn contract:compile && yarn contract:abi && yarn contract:seed",
+    "contract:abi": "node abi.js",
+    "contract:compile": "truffle compile --network test",
+    "contract:seed": "truffle test --compile-none --network test",
+    "codegen": "graph codegen",
+    "create:test": "graph create test/k6 --node $GRAPH_NODE_ADMIN_URI",
+    "deploy:test": "graph deploy test/k6 --version-label v0.0.1 --ipfs $IPFS_URI --node $GRAPH_NODE_ADMIN_URI"
+  },
+  "dependencies": {
+    "@graphprotocol/graph-cli": "0.29.0",
+    "@graphprotocol/graph-ts": "0.26.0",
```

Comment:
Do we want to pin those? I know the other `subgraphs` in the `integration-tests` folder mostly use the `main` branch of the repos.

Reply:
We have the lock files under source control so pinning won't affect it? I think we have dependabot configured for this repo so we'll updates through PRs?